inferno-ml-server/src/Inferno/ML/Server/Inference.hs

```diff
@@ -247,7 +247,12 @@ getParameter iid =
     =<< queryStore q (Only iid)
   where
     q :: Query
-    q = [sql| SELECT * FROM params WHERE id = ? LIMIT 1 |]
+    q =
+      [sql|
+        SELECT * FROM params
+        WHERE id = ? AND terminated IS NULL
+        LIMIT 1
+      |]
```

Comment:
I can't help mentioning https://hackage.haskell.org/package/postgresql-typed when I see quasi-quoted SQL intended for postgres :) It's the haskell equivalent of other "infer SQL types at compile time" libraries like https://ocaml.org/p/pgocaml_ppx/ or https://github.com/launchbadge/sqlx